### ISSUE TITLE
Usage of custom options

### ISSUE THREAD

**Issue by tscpp** [grpc/grpc-node#1772]
### Problem description
How do I get defined custom options from messages and services? The messages and service objects from output has `options` properties, but it is always null.

### Reproduction steps
1. Run `npm init -y` in a new directory
2. Run `npm install @grpc/proto-loader@0.6.1`
3. **main.js**
    ```javascript
    const proto = require('@grpc/proto-loader')
    const util = require('util')
    
    const definition = proto.loadSync('main.proto')
    console.log('MyService: ', util.inspect(definition.MyService, false, 1000, true))
    console.log('MyMessage: ', util.inspect(definition.MyMessage, false, 1000, true))
    ```
4. **main.proto**
    ```proto
    syntax = "proto3";
    import "google/protobuf/descriptor.proto";
    
    extend google.protobuf.MessageOptions {
        string my_option = 51234;
    }
    
    service MyService {
    	rpc MyRpc(MyMessage) returns (MyMessage) {
    		option (my_option) = "Hello world!";
    	}
    }
    
    message MyMessage {
        option (my_option) = "Hello world!";
    }
    ```
5. Run `node main.js`
6. The output doesn't contain any info about custom options.

### Environment
 - OS name, version and architecture: [Windows 10 64x 20H2]
 - Node version [v14.16.1]
 - Node installation method [website]
 - If applicable, compiler version []
 - Package name and version [@grpc/proto-loader@0.6.1]

#### Console output

GitHub Gist raw text: https://gist.githubusercontent.com/tscpp/32f889a1989b8f86e6a91af507438b53/raw/6973cab8d80dafcd2a94eff3ace1305ae133fd86/log.tx

**Comment by murgatroid99** [comment]
You are correct, `@grpc/proto-loader` does not currently output information about custom options.

**Comment by desy83** [comment]
Is there any progress on this feature request? Would be great to see it on the next Roadmap. Thanks

**Comment by desy83** [comment]
@murgatroid99 Is it possible to upvote custom message options? Thanks

**Comment by the-duke** [comment]
Are there any news to access custom service and method options via @grpc/proto-loader? Is this feature available in new protobufjs (7.xx) versions?
 @desytech: Do you have you found any workarounds to access options in node?

**Comment by n0v1** [comment]
Thanks for pointing at the [existing pull request](#2230). Let's continue the discussion there. I might find some time to work on this.

**Comment by n0v1** [comment]
Custom options **on message level** are supported as of [@grpc/proto-loader 0.7.13](https://github.com/grpc/grpc-node/releases/tag/%40grpc%2Fproto-loader%400.7.13).

**Comment by murgatroid99** [comment]
Version 0.7.13 added support for method options, not message options.
